include/spm.rst

```diff
@@ -14,13 +14,11 @@ Configuration
 *************
 
 Use Kconfig to configure the security attributions for the peripherals.
-Modify the source code of the SPM subsystem to configure the security attributions of flash or SRAM.
+Modify the source code of the SPM subsystem to configure the security attributions of SRAM.
```

Comment:
It's not just the source code anymore, right? Configuration is also via Kconfig, @lemrey @hakonfam ?

Reply:
Not for SRAM. The Kconfig is listed in the sentence above.